addons/website_event/static/src/scss/website_event.scss

```diff
@@ -1,13 +1,219 @@
-.o_wevent_participating_container {
-    position: relative;
+#o_we_search_box {
+    flex: 0 0 100%;
+    .form-control {
+        @include hover-focus {
+            border-color: $primary;
+            box-shadow: none;
+        }
+    }
+    @include media-breakpoint-up(lg) {
+       flex: 0 0 33%;
+    }
+}
+
+// TODO: Improve this part
+#o_we_index_topbar_filters {
+    .dropdown-toggle {
+        display: block;
+        border: 1px solid #ced4da;
+        padding: .5rem .75rem;
+        background-color: #fff;
+        @include border-radius($dropdown-border-radius);
+        color: #495057;
+        @include hover-focus {
+            border-color: $primary;
+            color: $primary;
```

Comment:
When you use a color use `theme-color('XXX')`, not `$XXX` (or `gray('XXX')` for grays)